### PR TITLE
[AIDAPP-280]: Improve the Service Request Updates on the Service Request page in the Portal

### PR DESCRIPTION
**Pull request by kandarp-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-280

### Technical Description

> Improve the Service Request Updates on the Service Request page in the Portal.

### Any deployment steps required?

> No.

### Are any Feature Flags and/or Data Migrations that can eventually be removed Added?

> No.

_______________________________________________

### CODE REVIEW

File: app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/GetServiceRequestsController.php [hunk 1]
```diff
@@ -117,8 +117,8 @@ public function show(ServiceRequest $serviceRequest): JsonResponse
                 ->serviceRequestUpdates()
                 ->latest('created_at')
                 ->where('internal', false)
-                ->get()
-                ->map(function (ServiceRequestUpdate $serviceRequestUpdate) {
+                ->paginate(10)
```
Comment (Orrison) [inline app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/GetServiceRequestsController.php:120]: Let's change this to `5` actually

File: app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/StoreServiceRequestUpdateController.php [hunk 2]
```diff
@@ -53,6 +54,22 @@ public function __invoke(StoreServiceRequestUpdateRequest $request): JsonRespons
...
         $serviceRequestUpdate->save();
 
-        return response()->json($serviceRequestUpdate, 201);
+        $serviceRequest = ServiceRequest::findOrFail($request->serviceRequestId);
+
+        $serviceRequestUpdates = $serviceRequest
+            ->serviceRequestUpdates()
+            ->latest('created_at')
+            ->where('internal', false)
+            ->paginate(10)
```
Comment (Orrison) [inline app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/StoreServiceRequestUpdateController.php:63]: Let's change this to `5` actually

File: portals/knowledge-management/src/Pages/ViewServiceRequest.vue
Comment (Orrison) [inline portals/knowledge-management/src/Pages/ViewServiceRequest.vue:1]: Right now changing the page in pagination seems to trigger a page reload. Is there a way to fix it so the data changes without the entire component reloading?

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/StoreServiceRequestUpdateController.php (hunk 2 above)
Comment (kandarp-canyon) [reply app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/StoreServiceRequestUpdateController.php]: done.

File: portals/knowledge-management/src/Pages/ViewServiceRequest.vue
Comment (kandarp-canyon) [reply portals/knowledge-management/src/Pages/ViewServiceRequest.vue:1]: Yes, I've applied condition for that to do not reload the page.

**Pull request merged**